pkg/crc/machine/kubeconfig.go

```diff
@@ -242,33 +242,42 @@ func contains(arr []string, str string) bool {
 	return false
 }
 
-func mergeKubeConfigFile(kubeConfigFile string) error {
-	globalConfigPath, kubeConf1, err := getGlobalKubeConfig()
-	if err != nil {
-		return err
+func mergeKubeConfigFile(kubeConfigFile, globalConfigFile string) error {
```

Comment:
Is this function signature only changed to have test around it?

Reply:
yes, I changed it because it was being used in just one place and so it can be tested more easily